From ce0ed4f45918f9230db435ee3b733090fa4f9620 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Tue, 27 Mar 2018 12:08:30 +0200
Subject: [PATCH] Remove historic _RPC_THREAD_SAFE_ preprocessor conditional

When the code was imported from glibc, the -D_RPC_THREAD_SAFE_
build flag was missed.
---
 rpcgen/rpc_main.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/rpcgen/rpc_main.c b/rpcgen/rpc_main.c
index 94bd818..1b26e52 100644
--- a/rpcgen/rpc_main.c
+++ b/rpcgen/rpc_main.c
@@ -1346,15 +1346,6 @@ parseargs (int argc, const char *argv[], struct commandline *cmd)
   cmd->Scflag = flag['C'];
   cmd->makefileflag = flag['M'];
 
-#ifndef _RPC_THREAD_SAFE_
-  if (mtflag || newstyle)
-    {
-      /* glibc doesn't support these flags.  */
-      f_print (stderr,
-	       _("This implementation doesn't support newstyle or MT-safe code!\n"));
-      return (0);
-    }
-#endif
   if (tirpcflag)
     {
       pmflag = inetdflag ? 0 : 1;    /* pmflag or inetdflag is always TRUE */
